app/controllers/application_controller.rb

```diff
@@ -1,4 +1,8 @@
 class ApplicationController < ActionController::API
+  include ActionController::HttpAuthentication::Token::ControllerMethods
+
+  respond_to :json
```

Comment:
I guess we want this to be here, since we are not going to support any other format here. If I don't this, devise response with html format by default, and ember does not really understand that